### PR TITLE
Allow to pass a default reciepient when creating a JWE

### PR DESCRIPTION
**Pull request by simo5**
Related to #53

### CODE REVIEW

**Comment by simo5** [comment]
@hannosch please take a look

File: jwcrypto/tests.py [hunk 1]
```diff
@@ -883,3 +883,9 @@ def test_jws_loopback(self):
         check.deserialize(o, jwk.JWK(kty='oct', k=base64url_encode(b'A' * 16)),
                           alg="HS512")
         self.assertTrue(check.objects['valid'])
+
+    def test_jwe_default_recipient(self):
+        key = jwk.JWK(kty='oct', k=base64url_encode(b'A' * (128 // 8)))
+        enc = jwe.JWE(plaintext='plain',
+                      protected='{"alg":"A128KW","enc":"A128GCM"}',
+                      recipient=key).serialize()
```
Comment (hannosch) [inline jwcrypto/tests.py]: Travis fails with a linting error, as ``enc`` is defined but not used.

File: jwcrypto/jwe.py [hunk 2]
```diff
@@ -842,7 +842,7 @@ class JWE(object):
     }
 
     def __init__(self, plaintext=None, protected=None, unprotected=None,
-                 aad=None, algs=None):
+                 aad=None, algs=None, recipient=None, header=None):
         """Creates a JWE token.
 
         :param plaintext(bytes): An arbitrary plaintext to be encrypted.
```
Comment (hannosch) [inline jwcrypto/jwe.py]: Shouldn't recipient and header be mentioned in the ``:param:`` docstring?

File: jwcrypto/tests.py (hunk 1 above)
Comment (simo5) [inline jwcrypto/tests.py]: Yeah fixed with a more complete test

File: jwcrypto/jwe.py (hunk 2 above)
Comment (simo5) [inline jwcrypto/jwe.py]: Yup, thanks for spotting this one

**Pull request closed without merging**